README.md

```diff
@@ -26,7 +26,14 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose(person) do
+
+@name = name
```

Comment:
Is there any need for this line? You already have access to the method argument with `person`. All you need to do now is print out it, as you do a few lines down.